src/opening-period/description/OpeningPeriodDescription.scss

```diff
@@ -0,0 +1,23 @@
+@import 'node_modules/hds-design-tokens/lib/all.scss';
+
+.add-new-opening-period-title {
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
+  font-size: $fontsize-body-l;
+  line-height: 2.5;
+  padding-left: $spacing-s;
+  padding-right: $spacing-s;
+  border: 2px solid rgb(128, 128, 128);
+  min-width: 400px;
+}
+
+.opening-period-optional-description {
+  padding: $spacing-layout-2-xs;
+  width: calc(0.5808333 * 100%);
+  height: 164px;
```

Comment:
Good call, I'll check if we could replace the height completely with the use of rows-attribute!

- [ ] Try to apply rows-attribute instead of hard-coding height

Reply:
Success, was able to successfully replace the height with rows attribute!